### ISSUE TITLE
Can't Update mPdf via composer

### ISSUE THREAD

**Issue by afsheen1** [mpdf/mpdf.github.io#148]
my previous version of mPdf was 8.0.5 so I run composer update to update all dependency.

but mpdf failed to update to 8.0.8 with following error.

`Failed to clone https://github.com/mpdf/mpdf.git, git was not found, check that it is installed and in your PATH env.  
                                                                                                                         
  'git' is not recognized as an internal or external command,                                                            
  operable program or batch file.`

**Comment by finwe** [comment]
Please note that the Issue tracker serves mainly as a place to report bugs and request new features as noted in [Contributing guidelines](https://raw.githubusercontent.com/mpdf/mpdf/development/.github/CONTRIBUTING.md). Please do not abuse it as a general questions or troubleshooting location.

Use the the [mpdf tag](https://stackoverflow.com/questions/tagged/mpdf) at [Stack Overflow](https://stackoverflow.com/).

**Comment by finwe** [comment]
always google your error message first.

https://stackoverflow.com/questions/17792588/composer-says-git-not-found

**Comment by afsheen1** [comment]
@finwe thank you for reply,

the error only occur with mpdf not any other dependencies. so that's why I post the issue here.

before posting the issue I Google the error, but none of them helped me...

I have Composer version 2+, and git version 2+, 

I already did `$ composer update/require mpdf/mpdf --prefer-dist/--prefer-source` but didn't worked. also I tried some other suggested options in stackoverflow.

I also cleared composer cache and tried again but didn't work. but I can install any package via same composer in same project. I have no idea the error only occur for me or some one else also.
